docs/sphinx/examples/python/providers/fermioniq.py

```diff
@@ -0,0 +1,25 @@
+import cudaq
+
+# You only have to set the target once! No need to redefine it
+# for every execution call on your kernel.
+cudaq.set_target(
+    "fermioniq", **{
```

Comment:
Could you please put `"fermioniq"` on the same line? We have a regex in `validate_container.sh` to parse Python target name in example files. Looks like it missed this one due to the line break.

Reply:
check